sympy/functions/combinatorial/factorials.py

```diff
@@ -255,7 +255,7 @@ def _eval(self, n):
 
     @classmethod
     def eval(cls, arg):
-        if arg.is_Number:
```

Comment:
Whoops, sorry - trying to get used to emacs. I did not notice that one. I'll re-factor this then.